configurablecare/app/src/main/java/com/google/fhir/examples/configurablecare/care/CareConfiguration.kt

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright 2022-2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.fhir.examples.configurablecare.care
+
+import ca.uhn.fhir.context.FhirContext
+import ca.uhn.fhir.context.FhirVersionEnum
+import com.google.gson.Gson
+import com.google.gson.JsonArray
+import org.hl7.fhir.r4.model.Resource
+
+val careConfig =
```

Comment:
Let's make this a JSON file in the assets folder?

Reply:
Done. Named the file care-config.json.